src/Data/Version/Haskell.purs

```diff
@@ -36,7 +36,7 @@ versionParser = do
   as <- nonNegativeInt `sepBy1` match' '.'
   bs <- option Nil (hyphen *> identifier `sepBy` hyphen)
   eof
-  pure $ Version as bs
+  pure $ Version (fromFoldable as) bs
```

Comment:
Non-empty combinators like `sepBy1` now return `NonEmptyList` as of:
https://github.com/purescript-contrib/purescript-parsing/pull/102

Reply:
Do you think we should consider updating the definition of `Version` here instead?
```
data Version = Version (NonEmptyList Int) (List String)
```

Reply:
The `sepBy1` means that the parser already can't ever produce a version with no integer components anyway, right?

Reply:
Yes, that’s probably the better idea. I forgot that the constructor isn’t exported

Reply:
The constructor in this module is exported, so it would be a breaking change (it's the other one whose constructor isn't exported). But this is kind of a breaking change anyway since we're effectively dropping support for pre 0.14.x versions, and I think this module is not used in many places anyway.